README.md

```diff
@@ -1,10 +1,10 @@
-redux-promise
-=============
+# redux-promise
 
-[![build status](https://img.shields.io/travis/acdlite/redux-promise/master.svg?style=flat-square)](https://travis-ci.org/acdlite/redux-promise)
+[![build status](https://img.shields.io/travis/redux-utilities/redux-promise/master.svg?style=flat-square)](https://travis-ci.org/redux-utilities/redux-promise)
+[![codecov](https://codecov.io/gh/redux-utilities/redux-promise/branch/master/graph/badge.svg)](https://codecov.io/gh/redux-utilities/redux-promise)
 [![npm version](https://img.shields.io/npm/v/redux-promise.svg?style=flat-square)](https://www.npmjs.com/package/redux-promise)
```

Comment:
- Can you remove `flat-square` style so we have it aligned like the others?
- Would you mind adding npm downloads/month badge too? :)

Reply:
Done